### ISSUE TITLE
Remove Twitter inline tip buttons

### ISSUE THREAD

**Issue by emerick** [brave/brave-browser#32107]
Due to changes in the Twitter API, we need to remove the Twitter inline tip buttons when browing Twitter.

**Comment by GeetaSarvadnya** [comment]
Verification **PASSED** on

```
Brave | 1.56.20 Chromium: 115.0.5790.171 (Official Build) (64-bit)
-- | --
Revision | cf9067bf10d8f798c24643029af1d24e275646d6-refs/branch-heads/5790@{#1924}
OS | Windows 10 Version 22H2 (Build 19045.3208)
```

Verified using dev component for:

```
Brave Local Data Updater - Version: 1.0.942
```
![image](https://github.com/brave/brave-browser/assets/38657976/31f7e2e5-f11b-47e5-87c6-b71fc4d0aa55)

Launched `1.56.20` using `--use-dev-goupdater-url` and ensured issue is fixed with the component `1.0.942`. Confirmed that there is no inline Tip icon for the twitter publishers

https://twitter.com/brave
https://twitter.com/bravelaurenwags

Example | Example
---------|---------
![image](https://github.com/brave/brave-browser/assets/38657976/6ae2ec05-d044-4876-8be3-1ab405a0c8d7) | ![image](https://github.com/brave/brave-browser/assets/38657976/074563f2-5beb-4c2b-a9e7-3a7ee02778a8)

**Comment by emerick** [comment]
This is in Greaselion `1.0.408`.
